web-console/5-components/order-from-catalog.md

```diff
@@ -114,19 +113,40 @@ Post 3.6, an additional Bind Step will be implemented.
 		- Mandatory fields should be indicated with an asterisk
 	- Optional "Show Advanced Options" link
 		- Available for some builder images
-	- Vertical scroll bar is displayed as needed, this scrolls the right side of the content
+	- Vertical scroll bar is displayed as needed
 - Buttons associated with this step
 	- Cancel, Back & Create buttons should be displayed
 	- Cancel should always be enabled
 	- Back should be enabled IF there is a previous Plan step
 	- Create should be disabled until all mandatory fields have been input
 
-### Step 4 - Results
+#### Form Layout
+- Labels are displayed before the input widget
+![template](img/configuration-step.png)
 
-#### Request has been submitted, and is in progress
-	** Mock needs to be updated **
+#### Form Layout - Mobile :iphone:
+- On mobile displays, labels are displayed above the input widgets
+	<img src="img/configuration-step-mobile.png" alt="alt text" width="200">
 
-![template](img/results-in-progress.png)
+#### Catalog item shows advanced options
+![template](img/config-step-hide-advanced.png)
```

Comment:
Should this and the images that follow be one- or two-column?

Reply:
should be one column ... had missed some images in my initial commit